nodebuilder/blob/cmd/blob.go

```diff
@@ -41,14 +41,26 @@ func init() {
 		&fee,
 		"fee",
 		-1,
-		"specifies fee for blob submission",
+		"specifies fee(in TIA) for blob submission",
 	)
 
 	submitCmd.PersistentFlags().Uint64Var(
 		&gasLimit,
 		"gas.limit",
 		0,
-		"specifies max gas for the blob submission",
+		"sets the maximum amount of gas that is allowed to consume during blob submission",
+	)
+
+	submitCmd.SetUsageTemplate("" +
+		"celestia blob submit [namespace] [blobData] [flags]\n\n " +
+		"Flags:\n" +
+		"      --fee       int      specifies fee(in TIA) for blob submission(optional)\n" +
+		"      --gas.limit uint64   sets the maximum amount of that gas is allowed to consume during blob submission(optional)\n" +
+		"      -h, --help           help for submit\n" +
+		"NOTE: fee and gas.limit params will be calculated automatically if they will not be provided.\n\n" +
```

Comment:
"...if they are not provided as arguments" might read better

Reply:
Thanks 🙏